docs/dashboards/data-quality-dashboard.asciidoc

```diff
@@ -67,6 +70,6 @@ You can share data quality results to help track your team's remediation efforts
 
 * Export results for one index:
 . Expand an index that has at least one incompatible field by clicking the arrow to the left of its *Result*.
-. From the *Summary* or *Incompatible fields* tabs, select *Add to new case* to open a new <<cases-overview,case>>, or click *Copy to clipboard* to copy a Markdown report to your clipboard.
+. From the *Summary*, *Incompatible fields*, or *Same family* tabs, select *Add to new case* to open a new <<cases-overview,case>>, or click *Copy to clipboard* to copy a Markdown report to your clipboard.
```

Comment:
The original statement appears to still be true, because all three of the tabs above include the `Copy to clipboard` button, however the `Same family` tab doesn't have an `Add to case` button per the screenshot below:

<img width="1574" alt="same_family_tab" src="https://github.com/elastic/security-docs/assets/4459398/9560aae2-4bb3-4967-8f5d-5f66e2288482">